source/code/include/scxsystemlib/scxostypeinfo.h

```diff
@@ -26,6 +26,8 @@
 
 namespace SCXSystemLib
 {
+    std::wstring OSTypeInfo_GetConfigPath();
```

Comment:
I'd say put this in the SCXProductDependencies namespace too, by doing something like this:

namespace SCXSystemLib 
{ 
    namespace SCXProductDependencies
    {
         std::wstring OSTypeInfo_GetConfigPath();
    }

    ....

Reply:
in getConfigPath() method, we have this line of code "return SCXSystemLib::OSTypeInfo_GetConfigPath();", there is not a space SCXProductDependencies in PAL.

Reply:
I will fix it.